bin/ns-status.js

```diff
@@ -41,8 +41,28 @@ function requireWithTimestamp (path) {
   return resolved;
 }
 
+function fixRecFlag (enacted) {
+  if (enacted) {
+    enacted.recieved = enacted.recieved || enacted.duration === enacted.requested.duration
```

Comment:
@scottleibrand, with this we don't need the hack in our ini's, removing that seems to have fixed the empty enacted.json filee